personal_assistant/main.py

```diff
@@ -78,6 +78,38 @@ def main():
         elif command == "entries":
             print(num_records(personal_assistant))
 
+        elif command == "add-note":
+            personal_assistant.add_note()
```

Comment:
another thing is - maybe we need to make sure we can't have empty tags like this:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/81c3f791-e12c-4a51-bb80-d9b14dace172)

Reply:
will be done on WE